forms-flow-theme/scss/_modal.scss

```diff
@@ -213,4 +213,201 @@
   max-width: 100%;
   margin: auto;
   
-}
\ No newline at end of file
+}
+
+//settings modal
+
+.settingsModal{
+  display: flex;
+flex-direction: column;
+align-items: flex-start;
+align-self: stretch;
+border-radius: var(--spacer-150);
+padding-top: 5rem;
+padding-bottom: 5rem;
+
+}
+
+
+
+
+
+.section{
+display: flex;
+padding: var(--spacer-200) var(--spacer-250) !important;
+flex-direction: column;
+align-items: flex-start;
+gap: var(--spacer-100);
+align-self: stretch;
+}
+
+.modalhr{
+border-bottom: 1px solid var(--ff-gray-000); 
+}
+
+
+.noteSection{
+display: flex;
+padding: var(--spacer-100);
+flex-direction: column;
+align-items: flex-start;
+gap: var(--spacer-050);
+align-self: stretch;
+border-radius: var( --radius-0100);
+background: var(--ff-gray-000);
+}
+
+.noteContent{
+align-self: stretch;
+color: var(--ff-gray-800, #303436);
+
+
+font-size: var(--font-size-xs);
+line-height: var(--text-line-height); 
+}
+
+.urlInput{
+display: flex;
+align-items: center;
+align-self: stretch;
+border-radius: var(--radius-0100);
+border: 1px solid var(--ff-base-600, #253DF4);
+}
+.settingsModal .list-group {
+  border: 1px solid var(--ff-base-600, #253DF4);
+  border-radius: var(--radius-0100);
+}
+
+.settingsModal .list-group-item {
+  border: none;
+}
+
+
+
+
+
+
+
+.fieldLabel{
+  font-size: var(--font-size-xs);
+}
+
+
+.input-drop-edit{
+  position: absolute;
+  top: 34.3rem; //to be adjusted
+  z-index: 100;
+  overflow: auto;
+  scrollbar-width: none;
+  border-radius: 16px;
+  border-bottom-right-radius: var(--radius-0100);
+  border-bottom-left-radius: var(--radius-0100);
+
+  border-bottom: 1px solid var(--base-base-60, #253DF4);
+
+}
+.input-drop-create{
+  position: absolute;
+  top: 47.2rem; //to be adjusted
+  z-index: 100;
+  height: 350px;
+  overflow: auto;
+  scrollbar-width: none;
+  border-radius: 16px;
+  border-bottom-right-radius: 16px;
+  border-bottom-left-radius: 16px;
+
+  border-bottom: 1px solid var(--ff-base-600, #253DF4);
+
+}
+
+.input-drop-view{
+  position: absolute;
+  top: 59.15rem; //to be adjusted
+  z-index: 100;
+  height: 350px;
+  overflow: auto;
+  scrollbar-width: none;
+  border-radius: var(--radius-0100);
+  border-bottom-right-radius: var(--radius-0100);
+  border-bottom-left-radius: var(--radius-0100);
+
+  border-bottom: 1px solid var(--ff-base-600, #253DF4);
+
+}
+
+.inputWithPills {
+  display: flex !important;
+  flex-wrap: wrap !important;
+  align-items: center;
+  padding: var(--spacer-050) var(--spacer-100);
+  border: 1px solid var(--ff-base-600, #253DF4) !important;
+  border-radius: var(--radius-0100) !important;
+  position: relative;
+  grid-row-gap: 5px !important;
+  row-gap: 5px !important;
+  grid-column-gap: 5px !important;
+  column-gap: 5px !important;
+
+.roleInput {
```

Comment:
can we use kebab casing for ALL css class names????

ex : role-input instead of roleInput ? 

@Bonymol-aot

Reply:
good catch

Reply:
casing updated